node/node_v2.go

```diff
@@ -134,3 +131,26 @@ func (n *Node) DownloadBundles(ctx context.Context, batch *corev2.Batch) ([]*cor
 
 	return blobShards, rawBundles, nil
 }
+
+func (n *Node) ValidateBatchV2(
```

Comment:
Should this just go in the core/validator object?

Reply:
I moved batch header validation as another method in validator object, but kept them separated